catalyst_voices/lib/widgets/common/resizable_box_parent.dart

```diff
@@ -0,0 +1,105 @@
+import 'dart:math';
+
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:flutter/widgets.dart';
+
+class ResizableBoxParent extends StatelessWidget {
```

Comment:
Add a line or two of docs what this widget does

Reply:
For private widgets it's not that needed but publicly widgets should be better documented.